client/common/directives/ModelField/ModelField.js

```diff
@@ -316,7 +316,18 @@ angular.module('dashboard.directives.ModelField', [
           if (!property.display.options) property.display.options = {};
           if (!property.display.options.format) property.display.options.format = "YYYY-MM-DD  h:mm A";
         }
-        
+
+        if (property.display.convertToLocalTime === false) {
+          // remove the 'Z' from the end of the timestamp so that it is not converted to local time
+          if (scope.data[scope.key]) {
+            scope.data[scope.key] = scope.data[scope.key].substring(0, scope.data[scope.key].length-1);
+          }
+        }
+
+        if (property.display.defaultValueUsingModelKey) {
```

Comment:
Can you do some testing around this? looks like you are forcing the default value even if the scope.data[scope.key] has a value already.